text/0044-heartbeat.md

```diff
@@ -0,0 +1,83 @@
+- Start Date: 2022-12-09
+- RFC Type: feature
+- RFC PR: N/A
+- RFC Status: Approved
+
+# Summary
+
+Through “heartbeat” monitoring, we want a way for the SDKs to communicate with Sentry so that they are at the bare minimum connected.
+
+This will also be expanded in the future to include richer diagnostic information about SDK configuration such as configured products (bug tracking, performance profiling, replay) or features (e.g. releases, customer sample rates, frameworks, integrations ) and potentially more.
```

Comment:
Very minor detail, but I would suggest to rephrase the configured products list to "error monitoring, performance monitoring, profiling, replays"